File: backend/src/main/java/org/example/LaptopStoreApplication.java
package org.example;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackages = {"org.example.*"})
@EntityScan(basePackages = {"org.example.model.entity"})
@EnableCaching
public class LaptopStoreApplication {
    public static void main(String[] args) {
        SpringApplication.run(LaptopStoreApplication.class);
    }
}

File: laptop-store-backend/src/main/kotlin/org/example/application/LaptopStoreApplication.kt
package org.example.application

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication(scanBasePackages = ["org.example.*"])
class LaptopStoreApplication

fun main(args: Array<String>) {
    runApplication<LaptopStoreApplication>(*args)
}
